src/service/service.ts

```diff
@@ -70,12 +70,12 @@ class Service {
 
   private validateTaskNames(){
     const nonDescribedTasks = Object.keys(this.tasks).filter(x => !this.definition.tasks[x]);
-    if (nonDescribedTasks.length > 0){
+    if (nonDescribedTasks.length > 0) {
       throw new Error(`The following tasks does not present in your mesg.yml: ${nonDescribedTasks.join(', ')}`);
     }
     const nonHandledTasks = Object.keys(this.definition.tasks).filter(x => !this.tasks[x]);
-    if (nonHandledTasks.length > 0){
-      throw new Error(`The following tasks described in your mesg.yml don't have callbacks: ${nonHandledTasks.join(', ')}`);
+    if (nonHandledTasks.length > 0) {
+      console.log(`WARNING: The following tasks described in your mesg.yml don't have callbacks: ${nonHandledTasks.join(', ')}`);
```

Comment:
Talking about callback is confusing I think.

```suggestion
      console.log(`WARNING: The following tasks described in your mesg.yml are not implemented: ${nonHandledTasks.join(', ')}`);
```

Reply:
done